data_models/install.sh

```diff
@@ -16,6 +16,8 @@ psql data_refinery -c 'ALTER USER data_refinery_user CREATEDB;'
 # See https://www.howtogeek.com/177621/the-beginners-guide-to-iptables-the-linux-firewall/
 # for more information.
 iptables -A INPUT -s 172.17.0.0/16 -j ACCEPT
+/sbin/iptables-save > /etc/iptables/rules
```

Comment:
Hmm that wasn't a good link to leave there. It was incomplete. I'll update it with a better one.